micro_dl/preprocessing/generate_masks.py

```diff
@@ -191,5 +191,5 @@ def generate_masks(self,
         meta_df = pd.concat([self.frames_metadata, mask_meta_df],
```

Comment:
Currently we are adding mask meta into the existing frames_meta.csv as a separate channel. Wondering if it would be simpler to save mask_meta by itself (to be similar to how you have it for mask_dir provided externally)?  no concat perhaps

Reply:
And then the concatenated dataframe is saved in mask_dir, which makes no sense at all...
I'll remove the concat and test.